### PR TITLE
Consistently use WordPress blue for links on table footers

### PR DESCRIPTION
**Pull request by elibud**
**Fixes** #7851 
And it also updates the AztecEditor footer to remove the info icon as agreed with @mattmiklic and to use WordPress blue in the link text.

**To test:**
Go to Jetpack Settings and check that the Learn More text is the correct color and that taping navigatest to the WebViewController.
Repeat for the Aztec editor feedback link on app settings.

simulator screen shot - iphone 8 - 2017-09-27 at 11 27 30simulator screen shot - iphone 8 - 2017-09-27 at 11 27 37

Needs review: @frosty

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift [hunk 1]
```diff
@@ -167,25 +165,26 @@ class AppSettingsViewController: UITableViewController {
...
-            view.tapBlock = { [weak self] in
-                WPAppAnalytics.track(.editorAztecBetaLink)
-
-                if let controller = self {
-                    WPWebViewController.presentWhatsNewWebView(from: controller)
-                }
-            }
-
-            return view
+            let footer = UITableViewHeaderFooterView.init(frame: CGRect(x: 0.0,
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift]: You can lose the `.init` here and just have `UITableViewHeaderFooterView(frame: ...)`

File: WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift [hunk 2]
```diff
@@ -167,25 +165,26 @@ class AppSettingsViewController: UITableViewController {
...
+                                                                        y: 0.0,
+                                                                        width: tableView.frame.width,
+                                                                        height: AppSettingsViewController.aztecEditorFooterHeight))
+            footer.textLabel?.text = NSLocalizedString("Editor release notes & bug reporting",
+                                                       comment: "Label for button linking to release notes and bug reporting help for the new beta Aztec editor")
+            footer.textLabel?.font = UIFont.preferredFont(forTextStyle: .footnote)
+            footer.textLabel?.isUserInteractionEnabled = true
+
+            let tap = UITapGestureRecognizer(target: self, action: #selector(self.handleEditorFooterTap(_:)))
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift]: I'd personally remove the `self.` in the `#selector`, as it's not needed.

File: WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift [hunk 3]
```diff
@@ -195,6 +194,11 @@ class AppSettingsViewController: UITableViewController {
         return section == Sections.editor.rawValue && EditorSettings().editor == .aztec
     }
 
+    func handleEditorFooterTap(_ sender: UITapGestureRecognizer) {
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift]: Let's make this `private` – but you'll also need to add `@objc` so the gesture recogniser can find it.

`@objc private func handleEditorFooterTap(...`

**Comment by elibud** [comment]
@frosty ready for another look!
Great catch on that background color. 
I've made the fixes you suggested on both `AppSettingsViewController` and `JetpackSecuritySettingsViewController`.
Thanks for the review :)

**Pull request merged**